Sources/Rendering/Core/ImageResliceMapper/test/testImageResliceMapperShareOpenGLTexture.js

```diff
@@ -63,13 +61,6 @@ test.onlyIfWebGL('Test ImageResliceMapperShareOpenGLTexture', (t) => {
   sslicePlane.setNormal(1, 0, 0);
   smapper.setSlicePlane(sslicePlane);
 
-  const oline = gc.registerResource(vtkOutlineFilter.newInstance());
-  const omapper = gc.registerResource(vtkMapper.newInstance());
-  omapper.setInputConnection(oline.getOutputPort());
-  const oactor = gc.registerResource(vtkActor.newInstance());
-  oactor.setMapper(omapper);
-  renderer.addActor(oactor);
-
```

Comment:
Why was the outline removed here?

Reply:
The outline filter generates axis-aligned bounding box which is not what we want for an oriented image. We could update the outline filter to handle vtkImageData differently from other vtkDataSets, but I wasn't sure if that is an expected behaviour of the `vtkOutlineFilter` (to generated an oriented box for Images). Hence, I took the shortcut of removing the outline since its not critical for the test.